stroyprombeton/admin/filters.py

```diff
@@ -25,19 +25,23 @@ def queryset(self, request, queryset):
         return queryset.filter(parent__slug=self.value())
 
 
-class CharacteristicsEqualityFilter(admin.SimpleListFilter):
+class DuplicateFilter(admin.SimpleListFilter):
 
-    title = _('characteristics equality')
-    parameter_name = 'equality'
+    title = _('duplicates')
+    parameter_name = 'duplicates'
 
     def lookups(self, request, model_admin):
         return (
-            ('h1', _('Only same h1')),
-            ('all', _('Same h1, characteristics, price'))
+            ('markh1', _('Mark and h1')),
+            ('all', _('Mark, code, price, specification and etc.'))
+            # @todo 591:30m Implement duplicates filter by tags and mark.
```

Comment:
```suggestion
            # @todo 591:60m Implement duplicates filter by tags, mark and series.
```

Every option has one related series.
Estimate is up to you